packages/platform_maps_flutter_google_android/lib/src/google_maps_widget.dart

```diff
@@ -0,0 +1,222 @@
+  ///
+  /// It is possible for other gesture recognizers to be competing with the map on pointer
+  /// events, e.g if the map is inside a [ListView] the [ListView] will want to handle
+  /// vertical drags. The map will claim gestures that are recognized by any of the
+  /// recognizers on this list.
+  ///
+  /// When this set is empty, the map will only handle pointer events for gestures that
+  /// were not claimed by any other gesture recognizer.
+  Set<Factory<OneSequenceGestureRecognizer>> get gestureRecognizers =>
+      params.gestureRecognizers;
+  @override
+  _PlatformMapState createState() => _PlatformMapState();
+}
+
+class _PlatformMapState extends State<_PlatformMap> {
+  @override
+  Widget build(BuildContext context) {
+    return google_map_original.GoogleMap(
+      initialCameraPosition:
+          widget.initialCameraPosition.googleMapsCameraPosition,
+      compassEnabled: widget.compassEnabled,
+      mapType: _getGoogleMapType(),
+      padding: widget.padding,
+      markers: widget.markers.googleMapsMarkerSet,
+      polylines: widget.polylines.googleMapsPolylineSet,
+      polygons: widget.polygons.googleMapsPolygonSet,
+      circles: widget.circles.googleMapsCircleSet,
+      gestureRecognizers: widget.gestureRecognizers,
+      onCameraIdle: widget.onCameraIdle,
+      myLocationButtonEnabled: widget.myLocationButtonEnabled,
+      myLocationEnabled: widget.myLocationEnabled,
+      onCameraMoveStarted: widget.onCameraMoveStarted,
+      tiltGesturesEnabled: widget.tiltGesturesEnabled,
+      rotateGesturesEnabled: widget.rotateGesturesEnabled,
+      zoomControlsEnabled: widget.zoomControlsEnabled,
+      zoomGesturesEnabled: widget.zoomGesturesEnabled,
+      scrollGesturesEnabled: widget.scrollGesturesEnabled,
+      onMapCreated: _onMapCreated,
+      onCameraMove: _onCameraMove,
+      onTap: _onTap,
+      onLongPress: _onLongPress,
+      trafficEnabled: widget.trafficEnabled,
+      minMaxZoomPreference:
+          widget.minMaxZoomPreference.googleMapsZoomPreference,
+    );
+  }
+
+  void _onMapCreated(google_map_original.GoogleMapController controller) {
+    widget.onMapCreated?.call(controller.platformMapController);
+  }
+
+  void _onCameraMove(google_maps.CameraPosition cameraPosition) {
+    widget.onCameraMove?.call(cameraPosition.platformCameraPosition);
+  }
+
+  void _onTap(google_maps.LatLng position) {
+    widget.onTap?.call(position.platformLatLng);
+  }
+
+  void _onLongPress(google_maps.LatLng position) {
+    widget.onLongPress?.call(position.platformLatLng);
+  }
+
+  google_maps.MapType _getGoogleMapType() {
+    if (widget.mapType == MapType.normal) {
+      return google_maps.MapType.normal;
+    } else if (widget.mapType == MapType.satellite) {
+      return google_maps.MapType.satellite;
+    } else if (widget.mapType == MapType.hybrid) {
+      return google_maps.MapType.hybrid;
+    }
+    return google_maps.MapType.normal;
+  }
```

Comment:
```suggestion
    switch (widget.mapType) {
        case MapType.normal:
          return google_maps.MapType.normal;
        case MapType.satellite:
          return google_maps.MapType.satellite;
        case MapType.hybrid:
          return google_maps.MapType.hybrid;
  }
```
I do not know if you want to apply some code changes already. In case here is one.